project/Dependencies.scala

```diff
@@ -178,6 +178,14 @@ object Dependencies {
       "org.mockito" % "mockito-core" % "2.7.11" % Test // MIT
     )
   )
+
+  val Kudu = Seq(
+    libraryDependencies ++= Seq(
+      "org.apache.kudu" % "kudu-client-tools" % "1.7.0", //Kudu
```

Comment:
Replace *Kudu* to *ApacheV2* as that is the license of kudu-client-tools as far as I can see.